challenges/security/application-security.md

```diff
@@ -0,0 +1,270 @@
+# Summary
+
+We use a synchronous, remote, one day hiring challenge for application
+security engineers. The challenge consists of the following sections:
+
+1. Kick off with the hiring manager. 45 minutes. 09:15 - 10:00 PST
+
+2. Peer review. You'll receive a brief Request for Discussion (RFD) and
+architecture diagram of a service. You will have 90 minutes to
+provide written PR feedback. 10:00 - 11:30 PST.
+
+3. After the peer review, section, there is a 30 minute break for lunch.
+
+4. Development. You'll receive prototype code for the service reviewed
+in part one. Building off this prototype codebase, we ask you develop
```

Comment:
"Part one" isn't defined. Should we say section 2? Or explicitly state it's different parts?

Reply:
Changed to "the service you reviewed earlier".